github/reactions.go

```diff
@@ -256,6 +256,101 @@ func (s ReactionsService) CreatePullRequestCommentReaction(ctx context.Context,
 	return m, resp, nil
 }
 
+// ListTeamDiscussionReactions lists the reactions for a team discussion.
+//
+// GitHub API docs: https://developer.github.com/v3/reactions/#list-reactions-for-a-team-discussion
+func (s *ReactionsService) ListTeamDiscussionReactions(ctx context.Context, teamID, discussionNumber int64, opt *ListOptions) ([]*Reaction, *Response, error) {
+	u := fmt.Sprintf("teams/%v/discussions/%v/reactions", teamID, discussionNumber)
+	u, err := addOptions(u, opt)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req.Header.Set("Accept", mediaTypeReactionsPreview)
+
+	var m []*Reaction
+	resp, err := s.client.Do(ctx, req, &m)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return m, resp, nil
+}
+
+// CreateTeamDiscussionReaction creates a reaction for a team discussion.
+//
+// GitHub API docs: https://developer.github.com/v3/reactions/#create-reaction-for-a-team-discussion
+func (s *ReactionsService) CreateTeamDiscussionReaction(ctx context.Context, teamID, discussionNumber int64, content string) (*Reaction, *Response, error) {
+	u := fmt.Sprintf("teams/%v/discussions/%v/reactions", teamID, discussionNumber)
+
+	body := &Reaction{Content: String(content)}
+	req, err := s.client.NewRequest("POST", u, body)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req.Header.Set("Accept", mediaTypeReactionsPreview)
+
+	m := &Reaction{}
+	resp, err := s.client.Do(ctx, req, m)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return m, resp, nil
+}
+
+//ListTeamDiscussionCommentReactions lists the reactions for a team discussion comment.
```

Comment:
In all the comments, please add a single space between the `//` and the first character (e.g. `// ListTeam...`) for consistency with the rest of the repo.

Reply:
Will do.